chrome-launcher/package.json

```diff
@@ -4,22 +4,25 @@
   "scripts": {
     "build": "tsc",
     "dev": "tsc -w",
-    "test": "mocha --require ts-node/register --reporter dot test/**/*-test.ts",
+    "test": "nyc mocha --require ts-node/register --reporter=dot test/**/*-test.ts --timeout=10000",
```

Comment:
I generally like to leave test as the regular command which makes coverage just `nyc npm test` or potentially `nyc npm test && nyc report`